openreview/conference/builder.py

```diff
@@ -951,15 +951,15 @@ def recruit_reviewers(self, invitees = [], title = None, message = None, reviewe
 
         invitation = self.invitation_builder.set_reviewer_recruiter_invitation(self, options)
         invitation = self.webfield_builder.set_recruit_page(self.id, invitation, self.get_homepage_options())
-        recruit_message = '''Dear {name},
 
-        You have been nominated by the program chair committee of ''' + self.short_name + ''' to serve as a reviewer.  As a respected researcher in the area, we hope you will accept and help us make the conference a success.
+        role = 'reviewer' if reviewers_name == 'Reviewers' else 'area chair'
```

Comment:
How do we do this in the request form when the reviewers_name is 'Area_Chairs'? Do we need to replace underscores with spaces?

Reply:
if reviewer_name == 'Area_Chairs', then role would be 'area chair', is this what you mean?

Reply:
I see, I think this is ok for this PR but we should translate the role from the group id so we can support other names than reviewers and area chairs.